.github/workflows/ci.yml

```diff
@@ -174,6 +174,8 @@ jobs:
         uses: davidB/rust-cargo-make@v1
         with:
           version: "0.36.4"
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v3
```

Comment:
Is there a way to not require additional dependency? For example, maybe `tonic` could use Rust-based implementation of the protobuf?

If not, I think we should follow @rymnc's suggestion and move Protobuf definition into a separate repository which will publish a Rust crate which we can use later in `fuel-core`, t avoid additional dependency for SDK and Sway

Reply:
Yeah. That makes sense to me.

Reply:
We need to remove dependency from `protoc` before we merge this change to the `master`=)